src/components/pages/Auth/Login/hooks-login/useRememberMe.ts

```diff
@@ -0,0 +1,63 @@
+import CryptoJS, { enc } from 'crypto-js';
+import { Dispatch, SetStateAction, useEffect } from 'react';
+
+type TUserCredentials = {
+  tag?: string;
+  password?: string;
+};
+
+type TRememberMeActions = {
+  rememberMe: boolean;
+  setTag: Dispatch<SetStateAction<string>>;
+  setPassword: Dispatch<SetStateAction<string>>;
+};
+
+type TRememberMeProps = TUserCredentials & TRememberMeActions;
+
+const key = import.meta.env.VITE_ENCRYPT_KEY;
+
+function saveUser({ tag, password }: TUserCredentials) {
+  const prevUser = localStorage.getItem('skoolizy-USER');
```

Comment:
The user doesn't need to be in caps. You can't just say skoolizy_user

Reply:
done